requirements.txt

```diff
@@ -1 +1,4 @@
 ops >= 1.2.0
+oci-image==1.0.0
+serialized-data-interface==0.2.0
```

Comment:
this is going to be changes to `>=` due to dep conflicts.

Reply:
Actually this will not work, because there is a max version `serialized-data-interface == 0.2.1`, which requires `ops=1.1`.
I will use:
```txt
serialized-data-interface == 0.2.1
ops == 1.1
oci-image == 1.0.0
minio == 7.0.3
```